CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
 cmake_minimum_required(VERSION 3.2)
 project(AwAssembler)
-add_subdirectory(src bin)
\ No newline at end of file
+
+set(libexec "lib/hinge")
+add_subdirectory(src bin)
+
+install(DIRECTORY scripts/ DESTINATION ${libexec})
```

Comment:
I should have added the argument `USE_SOURCE_PERMISSIONS` here so that the executables from `scripts/` can keep their executable bit when getting installed. See http://stackoverflow.com/a/13623977